src/plotman/plotman.py

```diff
@@ -159,8 +163,15 @@ def main():
         analyzer.analyze(args.logfile, args.clipterminals,
                 args.bytmp, args.bybitfield)
 
+    # 
+    # Exports log metadata to CSV
+    #
+    elif args.cmd == 'export_csv':
```

Comment:
What do you think of it just being an export and the presently implemented format is csv?

Reply:
Sounds good to me